packages/panzer/disc-fe/src/Panzer_IntegrationValues2.cpp

```diff
@@ -1135,7 +1135,8 @@ getWeightedMeasure(const bool cache,
       // Calculate measures (quadrature weights in physical space) for this side
       auto side_weighted_measure = Kokkos::DynRankView<Scalar,PHX::Device>("side_weighted_measure",num_evaluate_cells_,num_points_on_side);
       if(cell_dim == 1){
-        Kokkos::deep_copy(side_weighted_measure, side_cub_weights(0));
+        auto side_cub_weights_host = Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace(),side_cub_weights);
+        Kokkos::deep_copy(side_weighted_measure, side_cub_weights_host(0));
```

Comment:
This feels like a terribly inefficient way to accomplish this. How about instead of allocating and copying to the host, just

`Kokkos::parallel_for(policy, [](int i, int j) { side_weighted_measure(i, j) = side_cub_weights(0); } );`

Reply:
Though if I'm commenting on initialization code (I can't really tell), then whatever

Reply:
I will change that. Thanks!

Reply:
I wouldn't change it. What you do here with the deep_copy is more efficient.